README.md

```diff
@@ -1,56 +1,59 @@
 killbill-client-php
 ===================
 
-PHP client library for killbill
+PHP client library for [Killbill](http://killbill.io)
 
 Configuration
 -------------
 
 In order to user the library, you need to:
 
-1. Download the library and unzip it in your application
```

Comment:
I think just having the steps and commands to use composer in a non-composer project would be fine.